frontend_lib/src/component/Timeline/AutoCompleteTextArea.jsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react'
+import { commonMentionList } from '../../helper'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import Avatar from '../Avatar/Avatar'
+import AutoCompleteContainer from '../Input/AutoCompleteContainer/AutoCompleteContainer'
+
+export class AutoCompleteTextArea extends React.Component {
```

Comment:
Can we adapt this component to use at NewMemberForm.jsx too?

Reply:
We could, but I think the autoCompletion container for `NewMemberForm.jsx` is too different to have benefits, the data and the design is slightly different (mention != user).